javascript-modal/javascript-operators-and-expressions/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
Please remove the `javascript-operators-and-expressions` stuff from this solution.